.github/workflows/ci.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       uses: actions/setup-go@v5
       with:
         # renovate: datasource=golang-version depName=golang
-        go-version: 1.22.4
+        go-version: 1.23.2
```

Comment:
This should reference the go.mod file instead of a hardcoded version

Reply:
I agree with you, but the PR's purpose was not to change the mode and improve it but only to update the versions in the current configuration.